docs/Contributing/Pull Requests/pull_requests.md

```diff
@@ -0,0 +1,40 @@
+<a name="pull-requests"></a>
+## Pull requests
+
+Good pull requests - patches, improvements, new features - are great.
+
+**Please ask first** before embarking on any significant pull request (e.g.
+implementing new features, refactoring code, porting to a different language),
+otherwise you risk spending a lot of time working on something that the
+project's developers might not want to merge into the project. 
+
+[The discussions tab](https://github.com/RI-SE/ATOS/discussions) on the repo can be for this if you want to make sure that your endevour is something the maintainers consider relevant for the project.
+
+Please adhere to the coding conventions used throughout the project (indentation,
+accurate comments, etc.) and any other requirements (such as test coverage).
+
+Follow this process if you'd like your work considered for inclusion in the
+project.
+
+**IMPORTANT**: By submitting a patch, you agree to allow the project owner to
+license your work under the same license as that used by the project.
+
+
+#### Required information for a pull request
+
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+
+
+* **What is the current behavior?** (You can also link to an open issue here)
```

Comment:
This was a recommended step on a lot of open source projects to make it more clear what is actually being changed and if it actually is something we want to change. It might be a behavior that is actually ATOS working as intended. 

In the case of a bug the current behavior is the bug. I.e ATOS crashes when sending start to 5 objects etc.

Reply:
Ok that makes sense!